### PR TITLE
Remove schema requirements and validation from responses

### PR DESCRIPTION
**Pull request by omarkj**
This commit removes the schema requirement from
the API responses as they have been deprecated.

### CODE REVIEW

File: projector/src/main/java/com/projector/sdk/models/Envelope.java [hunk 1]
```diff
@@ -45,43 +42,32 @@
...
         return gson.toJson(this, objectType);
     }
 
     @Override
     public String toString() {
-        return "Envelope{" +
-                "schema='" + schema + '\'' +
-                ", data=" + data +
-                '}';
+        return String.format("Envelope{data=%s}", data);
```
Comment (trentonstrong) [inline projector/src/main/java/com/projector/sdk/models/Envelope.java]: hehe

Comment (trentonstrong) [inline projector/src/main/java/com/projector/sdk/models/Envelope.java]: Did you know javac automatically rewrites string concatenation of constants + variables to use `StringBuilder`? I agree this is more readable but as an FYI `java++`

Comment (omarkj) [inline projector/src/main/java/com/projector/sdk/models/Envelope.java]: I had no idea, no.

**Comment by trentonstrong** [comment]
I accept your PR with 👐

**Pull request merged**